src/utils.c

```diff
@@ -36,29 +36,50 @@ bool allNA(SEXP x) {
   // less space and time than any(is.na(x)) at R level because that creates full size is.na(x) first before any()
   // whereas this allNA can often return early on testing the first value without reading the rest
   const int n = length(x);
-  if (n==0) return true; // for empty raw, and NULL, to match R's all(is.na(raw())) true result, test 2116.13-14
+  if (n==0) // for empty raw, and NULL, to match R's all(is.na(raw())) true result, test 2116.13-14
+    return true;
   switch (TYPEOF(x)) {
   case LGLSXP:
   case INTSXP: {
     const int *xd = INTEGER(x);
-    for (int i=0; i<n; ++i)   if (xd[i]!=NA_INTEGER)   return false;
```

Comment:
@mattdowle sorry for removing your code alignment. I made it more friendly for codecov now.